molecule/default/group_vars/Pipfile

```diff
@@ -0,0 +1,11 @@
+[[source]]
```

Comment:
I think this file shouldn't be here, apart from that, looks good to me :)

Reply:
Done!